Code/GraphMol/MolAlign/AlignMolecules.cpp

```diff
@@ -69,6 +69,51 @@ double alignMol(ROMol &prbMol, const ROMol &refMol, int prbCid, int refCid,
   return res;
 }
 
+double getBestRMS(ROMol& refMol, ROMol& probeMol,
+                  int refId, int probeId,
+                  const std::vector<MatchVectType>& _matches)
+{
+  std::vector<MatchVectType> matches = _matches;
+  if (matches.empty()) {
+    bool uniquify = false;
+    bool recursionPossible = true;
+    bool useChirality = false;
+    bool useQueryQueryMatches = false;
+    int maxMatches = 1000;
```

Comment:
The 1000 here and the 1e6 below don't make sense together.
Should pick one and I think it makes sense to add it as an optional argument to the function.